src/components/Spinner.jsx

```diff
@@ -3,40 +3,37 @@ import React from "react";
 import classnames from "classnames";
 import PropTypes from "prop-types";
 
-const SPINNER_THEMES = {
-  dark: "dark",
-  light: "light",
-};
-
-const Spinner = ({ theme = "dark", className = "" }) => {
-  const spinnerItemClassName = classnames(
-    "neeto-ui-spinner__item",
-    {
-      "neeto-ui-bg-gray-800": theme === "dark",
-    },
-    {
-      "neeto-ui-bg-white": theme === "light",
-    }
-  );
+const SPINNER_THEMES = { dark: "dark", light: "light" };
 
-  return (
-    <span
-      className={classnames("neeto-ui-spinner", [className])}
-      data-testid="spinner"
-    >
-      <i className={spinnerItemClassName} />
-      <i className={spinnerItemClassName} />
-      <i className={spinnerItemClassName} />
-      <i className={spinnerItemClassName} />
-    </span>
-  );
-};
+const Spinner = ({ theme = "dark", size = 20, className = "" }) => (
```

Comment:
@goutham-subramanyam _a Let's restrict the size of the spinner component to two variants for now: `medium` size and `small`. Medium size should be the default size (`20px`). For small size variants, let's use `14px` for --neeto-ui-spinner-size.